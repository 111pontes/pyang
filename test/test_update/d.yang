module d {
  namespace urn:d;
  prefix d;

  import b {
    prefix b;
  }

  revision 2014-03-01;

  deviation /b:b/b:q {
    deviate add {
      default 'dummy';
    }
  }
}
