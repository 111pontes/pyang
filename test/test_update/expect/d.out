d@2014-04-01.yang:14: error: CHK_DEF_CHANGED
